daemon/options/bpf.go

```diff
@@ -31,7 +31,7 @@ type BpfConfig struct {
 	Cgroup2Path     string
 	EnableMda       bool
 	EnableBpfLog    bool
-	EnableAccesslog bool
+	EnableMetric    bool
```

Comment:
Does `enableMetric` control both metric and access log? `EnableMetric` -> `EnableObserve`？

Reply:
Done